docs/content/en/docs/getting-started/baremetal/bare-spec.md

```diff
@@ -261,6 +261,13 @@ spec:
 ### osFamily (required)
 Operating system on the machine. Permitted values: `bottlerocket`, `ubuntu`, `redhat` (Default: `bottlerocket`).
 
+### osImageURL
+Optional field to replace the default Bottlerocket operating system. EKS Anywhere can only auto-import Bottlerocket. In order to use Ubuntu or Redhat see [building baremetal node images]({{< relref "../../osmgmt/artifacts/#build-bare-metal-node-images" >}})
```

Comment:
Instead of "Redhat", I'd suggest RHEL or Red Hat Enterprise Linux.

Reply:
Good catch. i took this excerpt from the existing documentation for the Data center field. but i can correct this out in both the locations.!

Reply:
That would be great. Terminology can get really loose sometimes, so it's good to be consistent in how we refer to things when we can.